docs/integrations/ember-and-json-api.md

```diff
@@ -102,18 +118,16 @@ export default DS.JSONAPISerializer.extend({
 
 ```
 
+
 ## Including Nested Resources
```

Comment:
Would be great to mention the whole 'embedded resources' controversy here and how it's not JSON API so we don't do it, but ....